### PR TITLE
Added Documentation for python

### PR DESCRIPTION
**Pull request by Thiyagu55**
- Converted Hugo format docs from https://google.github.io/sqlcommenter/ into  markdown

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# opentelemetry-sqlcommenter
+SQLCommenter components for various languages
+
+Contains all the various `sqlcommenter-*` implementations.
+
+-  Python
+    -  [django](python/django/_index.md)
```
Comment (sjs994) [inline docs/README.md:7]: Would it be better to rename the file to `index.md` instead of `_index.md` ?

File: docs/python/SQLAlchemy/_index.md [hunk 2]
```diff
@@ -0,0 +1,204 @@
...
+When coupled with the web framework [flask](http://flask.pocoo.org), we still provide middleware to correlate
+your web applications with your SQL statements from psycopg2. Please see this end-to-end guide below:<br>
+[![](/docs/images/flask-logo.png)](/docs/python/flask/_index.md##with-psycopg2)
+
+### References
+
+Resource|URL
+---|---
+sqlcommenter-sqlalchemy on PyPi|https://pypi.org/project/google-cloud-sqlcommenter
+sqlcommenter-sqlalchemy on Github|https://github.com/google/sqlcommenter
```
Comment (sjs994) [inline docs/python/SQLAlchemy/_index.md:201]: Maybe we should point to the opentelemetry-sqlcommenter path ?

File: docs/python/django/_index.md [hunk 3]
```diff
@@ -0,0 +1,328 @@
...
+```
+## References 
+
+| Resource               | URL                                                                                                   |
+| ---------------------- | ----------------------------------------------------------------------------------------------------- |
+| Django                 | https://www.djangoproject.com/                                                                        |
+| OpenCensus             | https://opencensus.io/                                                                                |
+| opencensus-ext-django  | https://github.com/census-instrumentation/opencensus-python/tree/master/contrib/opencensus-ext-django |
+| sqlcommenter on PyPi   | https://pypi.org/project/google-cloud-sqlcommenter                                                    |
+| sqlcommenter on Github | https://github.com/google/sqlcommenter                                                                |
```
Comment (sjs994) [inline docs/python/django/_index.md:328]: opentelemetry-sqlcommenter ?

File: docs/python/psycopg2/_index.md [hunk 4]
```diff
@@ -0,0 +1,318 @@
...
+When coupled with the web framework [flask](http://flask.pocoo.org), we still provide middleware to correlate your web applications with your SQL statements from psycopg2. Please see this end-to-end guide below:<br>
+[![](/docs/images/flask-logo.png)](/docs/python/flask/_index.md##with-psycopg2)
+
+## References
+
+| Resource                        | URL                                                |
+| ------------------------------- | -------------------------------------------------- |
+| psycopg2 project                | http://initd.org/psycopg/docs/index.html           |
+| sqlcommenter-psycopg2 on PyPi   | https://pypi.org/project/google-cloud-sqlcommenter |
+| sqlcommenter-psycopg2 on Github | https://github.com/google/sqlcommenter             |
```
Comment (sjs994) [inline docs/python/psycopg2/_index.md:315]: opentelementry-sqlcommenter ?

File: docs/python/SQLAlchemy/index.md [hunk 5]
```diff
@@ -0,0 +1,204 @@
...
+`db_framework`|The version of SQLAlchemy in the form `'sqlalchemy:<sqlalchemy_version>'`|<div style="text-align: center">&#10060;</div>
+`traceparent`|The [W3C TraceContext.Traceparent field](https://www.w3.org/TR/trace-context/#traceparent-field) of the OpenCensus trace -- optionally defined with [`with_opencensus=True`](#with-opencensus)|<div style="text-align: center">&#10060;</div>
+`tracestate`|The [W3C TraceContext.Tracestate field](https://www.w3.org/TR/trace-context/#tracestate-field) of the OpenCensus trace -- optionally defined with [`with_opencensus=True`](#with-opencensus)|<div style="text-align: center">&#10060;</div>
+
+## End to end examples
+
+### Source code
+
+#### With OpenCensus
+```
```
Comment (aabmass) [inline docs/python/SQLAlchemy/index.md:92]:
```suggestion
```python
```

File: docs/python/SQLAlchemy/index.md [hunk 6]
```diff
@@ -0,0 +1,204 @@
...
+    with tracer.span():
+        result = engine.execute('SELECT * FROM polls_question')
+        for row in result:
+            print(row)
+
+if __name__ == '__main__':
+    main()
+```
+#### With DB Framework
+```
```
Comment (aabmass) [inline docs/python/SQLAlchemy/index.md:114]:
```suggestion
```python
```

File: docs/python/SQLAlchemy/index.md [hunk 7]
```diff
@@ -0,0 +1,204 @@
...
+    result = engine.execute('SELECT * FROM polls_question')
+    for row in result:
+        print(row)
+
+if __name__ == '__main__':
+    main()
+```
+
+#### With DB Driver
+```
```
Comment (aabmass) [inline docs/python/SQLAlchemy/index.md:137]:
```suggestion
```python
```

File: docs/python/django/index.md [hunk 8]
```diff
@@ -0,0 +1,332 @@
+layout: page
+title: "Django ReadMe"
+permalink: /python/django/index.md/
+
+# Django
+![](/docs/images/django-logo.png)
+
+<div class="row" >
+
+<div class = "column">
```
Comment (aabmass) [inline docs/python/django/index.md:10]: I don't think these are doing anything

File: docs/python/flask/index.md [hunk 9]
```diff
@@ -0,0 +1,150 @@
...
+
+| Field      | Included by default                            | Description                                        | Turn if off by          |
+| ---------- | ---------------------------------------------- | -------------------------------------------------- | ----------------------- |
+| controller | <div style="text-align: center">&#10004;</div> | The function being used to service an HTTP request | `with_controller=False` |
+| framework  | <div style="text-align: center">&#10004;</div> | "flask:<FLASK_VERSION>"                            | `with_framework=False`  |
+| route      | <div style="text-align: center">&#10004;</div> | The pattern used to match an HTTP request          | `with_route=False`      |
+
+## End to end examples
+### With psycopg2
+```
```
Comment (aabmass) [inline docs/python/flask/index.md:45]:
```suggestion
```python
```

File: docs/python/flask/index.md [hunk 10]
```diff
@@ -0,0 +1,150 @@
...
+2019-06-08 12:19:11.284 PDT [70984] LOG:  statement: SELECT * FROM polls_question
+/*controller='get_polls',db_driver='psycopg2',framework='sqlalchemy%3A1.3.4',
+route='/polls',traceparent='00-5b3df77064f35f091e89fb40022e2a1d-9bbd4868cf0ba2c3-01'*/
+```
+
+### With sqlalchemy
+
+Having successfully installed [google-cloud-sqlcommenter](/docs/python/SQLAlchemy/index.md) you can now just run
+
+```
```
Comment (aabmass) [inline docs/python/flask/index.md:103]:
```suggestion
```python
```

File: docs/python/psycopg2/index.md [hunk 11]
```diff
@@ -0,0 +1,318 @@
...
+| `controller` | Grabs the controller being used     | `with_controller`     |
+| `framework`  | Grabs the framework and its version | `with_framework`      |
+| `route`      | Grabs the route being used          | `with_route`          |
+
+## End to end examples
+
+### Source code
+
+#### With OpenCensus
+```
```
Comment (aabmass) [inline docs/python/psycopg2/index.md:113]:
```suggestion
```python
```

File: docs/python/psycopg2/index.md [hunk 12]
```diff
@@ -0,0 +1,318 @@
...
+                cursor.execute("SELECT * FROM polls_question")
+                for row in cursor:
+                    print(row)
+
+if __name__ == '__main__':
+    main()
+```
+
+#### With DB Driver
+```
```
Comment (aabmass) [inline docs/python/psycopg2/index.md:147]:
```suggestion
```python
```

File: docs/python/psycopg2/index.md [hunk 13]
```diff
@@ -0,0 +1,318 @@
...
+            cursor.execute("SELECT * FROM polls_question")
+            for row in cursor:
+                print(row)
+
+if __name__ == '__main__':
+    main()
+```
+
+#### With DB API Level
+```
```
Comment (aabmass) [inline docs/python/psycopg2/index.md:169]:
```suggestion
```python
```

File: docs/python/psycopg2/index.md [hunk 14]
```diff
@@ -0,0 +1,318 @@
...
+            cursor.execute("SELECT * FROM polls_question")
+            for row in cursor:
+                print(row)
+
+if __name__ == '__main__':
+    main()
+```
+
+#### With DB API Thread Safety
+```
```
Comment (aabmass) [inline docs/python/psycopg2/index.md:191]:
```suggestion
```python
```

File: docs/python/psycopg2/index.md [hunk 15]
```diff
@@ -0,0 +1,318 @@
...
+            cursor.execute("SELECT * FROM polls_question")
+            for row in cursor:
+                print(row)
+
+if __name__ == '__main__':
+    main()
+```
+
+#### With Driver Parameter Style
+```
```
Comment (aabmass) [inline docs/python/psycopg2/index.md:213]:
```suggestion
```python
```

File: docs/python/psycopg2/index.md [hunk 16]
```diff
@@ -0,0 +1,318 @@
...
+            cursor.execute("SELECT * FROM polls_question")
+            for row in cursor:
+                print(row)
+
+if __name__ == '__main__':
+    main()
+```
+
+#### With libpq Version
+```
```
Comment (aabmass) [inline docs/python/psycopg2/index.md:235]:
```suggestion
```python
```

**Review by aabmass — APPROVED** [review]
I didn't look too closely at all of the wording, but LGTM. Thanks!

File: docs/python/SQLAlchemy/_index.md (hunk 2 above)
Comment (Thiyagu55) [reply docs/python/SQLAlchemy/_index.md]: Updated

File: docs/python/django/_index.md (hunk 3 above)
Comment (Thiyagu55) [reply docs/python/django/_index.md]: Updated

File: docs/python/psycopg2/_index.md (hunk 4 above)
Comment (Thiyagu55) [reply docs/python/psycopg2/_index.md]: Updated

File: docs/python/django/index.md (hunk 8 above)
Comment (Thiyagu55) [reply docs/python/django/index.md]: Removed it

File: docs/python/SQLAlchemy/README.md [hunk 17]
```diff
@@ -0,0 +1,205 @@
...
+When coupled with the web framework [flask](http://flask.pocoo.org), we still provide middleware to correlate
+your web applications with your SQL statements from psycopg2. Please see this end-to-end guide below:<br>
+[![](../../images/flask-logo.png)](../flask/#with-psycopg2)
+
+### References
+
+Resource|URL
+---|---
+sqlcommenter-sqlalchemy on PyPi|https://pypi.org/project/google-cloud-sqlcommenter
+sqlcommenter-sqlalchemy on Github|https://github.com/open-telemetry/opentelemetry-sqlcommenter
```
Comment (sjs994) [inline docs/python/SQLAlchemy/README.md:202]: Can you add link these addresses

File: docs/python/django/README.md [hunk 18]
```diff
@@ -0,0 +1,314 @@
...
+> source <br>
+```
+git clone https://github.com/open-telemetry/opentelemetry-sqlcommenter.git
+cd python/sqlcommenter-python && python3 setup.py install
+```
+
+### Enabling it
+
+Please edit your `settings.py` file to include `google.cloud.sqlcommenter.django.middleware.SqlCommenter` in your `MIDDLEWARE` section like this:
+```diff python
```
Comment (sjs994) [inline docs/python/django/README.md:75]: i don't think \`\`\`diff python... ``` would work.

It's breaking in the github-pages. Maybe for this we should only use \`\`\`diff ... ``` for this

File: docs/python/django/README.md [hunk 19]
```diff
@@ -0,0 +1,314 @@
...
+```
+## References 
+
+| Resource               | URL                                                                                                   |
+| ---------------------- | ----------------------------------------------------------------------------------------------------- |
+| Django                 | https://www.djangoproject.com/                                                                        |
+| OpenCensus             | https://opencensus.io/                                                                                |
+| opencensus-ext-django  | https://github.com/census-instrumentation/opencensus-python/tree/master/contrib/opencensus-ext-django |
+| sqlcommenter on PyPi   | https://pypi.org/project/google-cloud-sqlcommenter                                                    |
+| sqlcommenter on Github | https://github.com/open-telemetry/opentelemetry-sqlcommenter                                                                |
```
Comment (sjs994) [inline docs/python/django/README.md:314]: Maybe we should add link the urls: Does this work: <https://www.markdownguide.org/basic-syntax/#urls-and-email-addresses>

File: docs/python/django/aws.md [hunk 20]
```diff
@@ -0,0 +1,45 @@
...
+
+>If any middleware execute database queries (that you'd like commented by SqlCommenter), those middleware MUST appear after
+'google.cloud.sqlcommenter.django.middleware.SqlCommenter'
+
+## References
+
+| Resource                             | URL                                                                        |
+| ------------------------------------ | -------------------------------------------------------------------------- |
+| Deploying Python applications on AWS | https://aws.amazon.com/getting-started/projects/deploy-python-application/ |
+| General sqlcommenter Django guide    | [link](README.md)                                                     |
```
Comment (sjs994) [inline docs/python/django/aws.md:45]: we can use this syntax for the full url here: <https://www.markdownguide.org/basic-syntax/#urls-and-email-addresses>.

File: docs/python/django/gcp.md [hunk 21]
```diff
@@ -0,0 +1,45 @@
...
+## Introduction
+
+This guide will help you add [sqlcommenter](https://github.com/open-telemetry/opentelemetry-sqlcommenter) to your Django applications running on [Google Cloud Platform (GCP)](https://cloud.google.com)
+
+## Requirements
+
+| Steps                     | Resource                                             |
+| ------------------------- | ---------------------------------------------------- |
+| Django on GCP             | https://cloud.google.com/python/django/              |
+| google-cloud-sqlcommenter | https://pypi.org/project/google-cloud-sqlcommenter   |
```
Comment (sjs994) [inline docs/python/django/gcp.md:20]: full url syntax for example: 

```
<https://cloud.google.com/python/django/>
```

File: docs/python/django/aws.md [hunk 22]
```diff
@@ -0,0 +1,45 @@
...
+## Introduction
+
+This guide will help you add [sqlcommenter](https://github.com/open-telemetry/opentelemetry-sqlcommenter) to your Django applications running on [Amazon Web Services (AWS)](https://aws.amazon.com)
+
+## Requirements
+
+| Steps                     | Resource                                                                   |
+| ------------------------- | -------------------------------------------------------------------------- |
+| Python on AWS             | https://aws.amazon.com/getting-started/projects/deploy-python-application/ |
+| google-cloud-sqlcommenter | https://pypi.org/project/google-cloud-sqlcommenter                         |
```
Comment (sjs994) [inline docs/python/django/aws.md:20]: Please encode the urls with 
```
<url>
```

File: docs/python/django/local.md [hunk 23]
```diff
@@ -0,0 +1,51 @@
...
+
+This guide will help you add [sqlcommenter](https://github.com/open-telemetry/opentelemetry-sqlcommenter) to your Django applications running locally.
+
+Please see the reference for the fields added in the SQL comments [google-cloud-sqlcommenter.Fields](README.md/#fields)
+
+## Requirements
+
+| Steps                     | Resource                                             |
+| ------------------------- | ---------------------------------------------------- |
+| Django                    | https://docs.djangoproject.com/en/stable/intro/      |
```
Comment (sjs994) [inline docs/python/django/local.md:22]: Please add syntax for full urls

File: docs/python/django/local.md [hunk 24]
```diff
@@ -0,0 +1,51 @@
...
+
+>If any middleware execute database queries (that you'd like commented by SqlCommenter), those middleware MUST appear after
+'google.cloud.sqlcommenter.django.middleware.SqlCommenter'*
+
+
+## References
+
+| Resource                     | URL                                              |
+| ---------------------------- | ------------------------------------------------ |
+| Django quickstart            | https://docs.djangoproject.com/en/stable/intro/  |
```
Comment (sjs994) [inline docs/python/django/local.md:50]: syntax for full urls

**Review by sjs994 — COMMENTED** [review]
Thanks Thiyagu!

Few minor comments to link the full urls.

**Review by sjs994 — APPROVED** [review]
Looks good to me

**Pull request merged**